cli/cmd/cluster.go

```diff
@@ -356,7 +354,7 @@ func newGetCredentialsCommand() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&opts.serviceAccountName, "service-account-name", defaultServiceAccountName, "the name of the service account")
-	cmd.Flags().StringVar(&opts.serviceAccountNamespace, "service-account-namespace", defaultServiceAccountNs, "the namespace in which the service account will be created")
+	cmd.Flags().StringVar(&opts.serviceAccountNamespace, "service-account-namespace", defaultMulticlusterNamespace, "the namespace in which the service account will be created")
```

Comment:
Actually the NS is hard-coded to `defaultMulticlusterNamespace` above where the Secret is created, and this param is for the NS of the SA in the remote cluster.